apps/dot_ext/tests/test_views.py

```diff
@@ -141,13 +159,19 @@ def test_delete_token_success(self):
             'an app', grant_type=Application.GRANT_AUTHORIZATION_CODE,
             redirect_uris='http://example.it')
         application.scope.add(capability_a)
-        tkn = self._create_test_token(anna, application)
-        response = self.client.delete(reverse('token_management:token-detail', args=[tkn.pk]),
+        self._create_test_token(anna, application)
```

Comment:
I want to update this test with two assertions:
1. another token for a different user but the same app continues to work.
2. multiple tokens associated with the revoked grant fail to work.